2-mandatory/2-retrying-random-numbers.js

```diff
@@ -5,20 +5,26 @@
 
 // This function shouldn't be changed
 function generateRandomNumber() {
-    console.log("Generating number...");
-    return Math.round(Math.random() * 100);
+  console.log("Generating number...");
+  return Math.round(Math.random() * 100);
 }
 
 function getRandomNumberGreaterThan50() {
-    // TODO - implement using a do-while loop
+  // TODO - implement using a do-while loop
+  let count;
+
+  do {
+    count = generateRandomNumber();
+  } while (count <= 50);
+  return count;
 }
 
```

Comment:
Good use of a do while loop to make the code clear and simple

Reply:
thank you